robot/client/client.go

```diff
@@ -1215,7 +1215,7 @@ func (rc *RobotClient) Tunnel(ctx context.Context, conn io.ReadWriteCloser, dest
 			// Close the channel first so that network errors can be filtered
 			// and prevented in the RecvWriterLoop.
 			close(rsDone)
-			readerSenderErr = errors.Join(readerSenderErr, client.CloseSend())
+			utils.UncheckedError(client.CloseSend())
```

Comment:
@cheukt one more swallow: `client.CloseSend` can also fail with `io.ErrClosedPipe`, and, per other closing logic, I think we should ignore it.